lib/slave/slave.go

```diff
@@ -11,27 +11,31 @@ import (
 
 var slaveInstance Slave
 
+type task struct {
+	lib.Task
+	Status       lib.Status
+	Password     string
+	progressChan chan chan string
+}
+
 type Slave struct {
+	id              string
```

Comment:
We probably don't need `id` here. Same for the `Init` function, the id is always "instance.EC2.cong1".